src/main/resources/se/diabol/jenkins/pipeline/DeliveryPipelineView/RegExpSpec/help-showUpstream.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    Will show upstream jobs for the first job.
```

Comment:
@KostyaSha ok. Must have gotten lost somewhere. Hope you got your answers already?